experiment/posttest.json

```diff
@@ -1,18 +1,39 @@
-[
-{"question":"This is a Sample Question 1?",
-"answers":{
-            "a":"answer1",
-            "b":"answer2",
-            "c":"answer3",
-            "d":"answer4"
-          },
-"correctAnswer":"a"},
-{"question":"This is a Sample Question 2?",
-"answers":{
-            "a":"answer1",
-            "b":"answer2",
-            "c":"answer3",
-            "d":"answer4"
-          },
-"correctAnswer":"c"} 
-]
+{
+	"version": 2.0,
+	"questions": [
+		{
+			"question": "1. A clay soil sample is tested in a triaxial apparatus in consolidated drained conditions at a cell pressure of 100 kN/m<sup>2</sup>. What will be the pore water pressure at a deviator stress of 40 kN/m<sup>2</sup>? ",
+			"answers": {
+				"a": "0 kN/m<sup>2</sup>",
+				"b": "20 kN/m<sup>2</sup>",
+				"c": "40 kN/m<sup>2</sup>",
+				"d": "60 kN/m<sup>2</sup>"
+			},
+			"explanations": {
+				"a": "In consolidated and drained conditions pore pressure is zero.",
```

Comment:
These explanations are not adding any value to the question. They are simply giving away the answer. Let's remove them.